src/main/scala/io/github/spark_redshift_community/spark/redshift/RedshiftWriter.scala

```diff
@@ -326,7 +326,7 @@ private[redshift] class RedshiftWriter(
         s"""{"url":"$sanitizedTempDir/$file", "mandatory":true}"""
       }
       val manifest = s"""{"entries": [${manifestEntries.mkString(",\n")}]}"""
-      val manifestPath = sanitizedTempDir + "/manifest.json"
+      val manifestPath = tempDir + "manifest.json"
```

Comment:
```
      // It's possible that tempDir contains AWS access keys. We shouldn't save those credentials to
// S3, so let's first sanitize `tempdir` and make sure that it uses the s3:// scheme:
```

You should still use a sanitized version of the url here. Also it'd  be useful to the reviewers if you can print out the detailed outcome of your tests (manual and automatic)

Please ask @88manpreet or someone else to review, as I am not a contributor to this anymore.

Reply:
I think that your statement is not correct, the `sanitizedTempDir` is necessary for the `COPY` command and that part is still as before (see L326).

@88manpreet can you please review it? Thank you so much!

Reply:
The manifestPath is the return value of `unloadData` and eventually feeds into `RedshiftWriter.copySql` via `RedshiftWriter.doRedshiftLoad`, which itself does `val fixedUrl = Utils.fixS3Url(manifestUrl)`.  So I think this change is a no-op except for it not stripping out the credentials now?  

And thematically, the manifestPath gets passed into the Redshift Copy command, redshift would want the s3:// format.

Have you tested that this change solves your original problem?

Reply:
Perhaps the change we need is that `val fsDataOut = fs.create(new Path(manifestPath))` needs to be `val fsDataOut = fs.create(new Path(tempDir))`, though we probably still want to strip the credentials out of tempDir first.  I think we would want to leave the return value unchanged though?